src/main/groovy/com/avioconsulting/mule/linter/model/MuleComponent.groovy

```diff
@@ -19,6 +26,10 @@ class MuleComponent {
         return attributes.get(name)?.length() > 0
     }
 
+    String getComponentName() {
+        return componentName
+    }
+
     String getName() {
```

Comment:
Name was removed above, why do the getter and setter still exist?

Reply:
removed now, forgot to take it off.